### PR TITLE
Another method to approve client upload

### PR DESCRIPTION
**Pull request by anisimkov**
Better fit for HTTP/2 implementation.

Part of S507-051.

### CODE REVIEW

File: src/core/aws-response.ads [hunk 1]
```diff
@@ -272,6 +272,22 @@ package AWS.Response is
...
+   --  It is to control the client data upload. If upload data size is known
+   --  from Content-Length header and less than Upload_Size_Limit configuration
+   --  parameter than the client message body arrived at once to the dispatcher
+   --  handler. User can understand it with Is_Body_Uploaded call from package
+   --  AWS.Status. If upload data size is unknown or more than
+   --  Upload_Size_Limit than the Is_Body_Uploaded returns False and user is
+   --  able to allow or disable the client data upload. If user returns
+   --  Continue response from dispatcher handler, then next time the dispatcher
+   --  handler will be called with uploaded body from client. If user returns
+   --  some another responce than client body upload will be terminated and
```
[the remark this answers is not in the document]
Comment (anisimkov) [reply src/core/aws-response.ads]: 2 "than" corrected to "then"

File: src/core/aws-response.ads [hunk 2]
```diff
@@ -272,6 +272,22 @@ package AWS.Response is
    --  with the request. This is used by the callback's chain in the
    --  dispatchers and should not be used by users.
 
+   function Continue return Data with
+     Post => Status_Code (Continue'Result) = Messages.S100
+             and then Mode (Continue'Result) = No_Data;
+   --  Returns an empty message (Data_Mode = No_Data and Status_Code is 100).
```
Comment (TurboGit) [inline src/core/aws-response.ads:278]: I'm proposing this text:

```
--  Returns an empty message (Data_Mode = No_Data and Status_Code is 100).
--  It is to control the client data upload.
--
--  If upload data size is known from Content-Length header and less
--  than Upload_Size_Limit configuration parameter then the client
--  message body arrived at once to the dispatcher handler. User can
--  check this by calling AWS.Status.Is_Body_Uploaded.
--
--  If upload data size is unknown or more than Upload_Size_Limit then
--  Is_Body_Uploaded returns False and user is able to allow or
--  disable the client data upload.

--  If user returns Continue response from dispatcher handler, then
--  next time the dispatcher handler will be called with uploaded body
--  from client. If user returns some other responses then client body
--  upload will be terminated and ignored.
```

**Review by TurboGit — COMMENTED** [review]
A proposal for the comment. Apart from that it looks good.

File: src/core/aws-response.ads (hunk 2 above)
Comment (anisimkov) [reply src/core/aws-response.ads:278]: Done

**Review by TurboGit — APPROVED** [review]
Thanks

**Pull request merged**